mbus-api/src/v0.rs

```diff
@@ -953,3 +962,16 @@ pub struct RemoveVolumeNexus {
     pub node: Option<NodeId>,
 }
 bus_impl_message_all!(RemoveVolumeNexus, RemoveVolumeNexus, (), Volume);
+
+/// Generic JSON gRPC request
+#[derive(Serialize, Deserialize, Default, Debug, Clone)]
+#[serde(rename_all = "camelCase")]
+pub struct JsonGrpcRequest {
+    /// id of the mayastor instance
+    pub node: NodeId,
+    /// JSON gRPC method to call
+    pub method: JsonGrpcMethod,
+    /// parameters to be passed to the above method
+    pub params: JsonGrpcParams,
+}
+bus_impl_message_all!(JsonGrpcRequest, JsonGrpcRequest, String, JsonGrpc);
```

Comment:
Is the reply always in json format? If so should it be serde_json::value::Value instead of String?

Reply:
It is returned from Mayastor as a "prettified" string.